content/project/teams/hosting.adoc

```diff
@@ -0,0 +1,85 @@
+---
+layout: project
+title: "Hosting Team"
+section: projects
+tags:
+- hosting
+- plugin-site
+links:
+  mailinglist: "/mailing-lists/#jenkinsci-dev-googlegroups-com"
+---
+
+Jenkins Hosting Team handles various requests related to hosting of open-source plugins and other components. (forking and transferring plugins, managing permissions and update centers, etc.).
+There are multiple contributors involved in this activity on regular basis.
+
+== Team responsibilities
+
+* Triage and processing of new plugin HOSTING requests in Jenkins Jira.
```

Comment:
Maybe put a link to the HOSTING project in Jira?

Reply:
```suggestion
* Triage and processing of new plugin link:https://issues.jenkins-ci.org/projects/HOSTING[HOSTING] requests in Jenkins Jira.
```